<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLI</class>
 <widget class="QWidget" name="CLI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
	background-image: url(:/back/views/hackback.jpg);

}
QFrame{
background:rgba(0,0,0,0.8);
border-radius:15px;
}

QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
font-size:22px

}
QPushButton:hover{
color: #03a9f4;
background-color:#fff;
}
QLabel{
color:#03a9f4;
background:transparent;
font-size:30px;

}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>991</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Form{
background :#333;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>WELCOME TO MARKOV PASSWORD GENERATOR!</string>
    </property>
   </widget>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>60</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton{
background:#48C9B0;
font-family:Ariel;
font-size:17px;
}</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>60</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton{
background:#48C9B0;
font-family:Ariel;
font-size:17px;

}</string>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="backimage.qrc"/>
 </resources>
 <connections/>
</ui>
